src/objecttypes/tests/test_version_api.py

```diff
@@ -0,0 +1,105 @@
+from datetime import date
+
+from django.urls import reverse
+
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from objecttypes.core.constants import ObjectVersionStatus
+from objecttypes.core.models import ObjectVersion
+from objecttypes.core.tests.factories import ObjectTypeFactory, ObjectVersionFactory
+from objecttypes.utils.test import TokenAuthMixin
+
+JSON_SCHEMA = {
+    "type": "object",
+    "title": "Tree",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "required": ["diameter"],
+    "properties": {"diameter": {"type": "integer", "description": "size in cm."}},
+}
+
+
+class ObjectVersionAPITests(TokenAuthMixin, APITestCase):
+    def test_get_versions(self):
```

Comment:
Can you add a test where there are 2 Objecttypes, with each 1 version. Then, check that if you retrieve `/objecttypes/<uuid>/versions` and the inline version URLs in `/objecttypes/<uuid>` only contain 1 version (and not also the version object from the other objecttype).

I had this issue quite often when working with nested viewsets.

Reply:
`test_get_objecttypes_with_versions` is added